notebooks/demo2/time-anlysis.ipynb

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
<blockquote>70% of the pdfs will take a few secs to 24 mins</blockquote><p>will this be less than 16 mins or less? since 75% of pdfs take less than 17.4 mins?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>This is according the standard deviation;  avg +- std (0-24min) covers ~70% of the normal distribution. There's the assumption though, that the times are in normal distribution. But yeah the more exact statement would be if take 70th or 75th percentile. </p>